source/includes/common-minio-tiering.rst

```diff
@@ -0,0 +1,97 @@
+.. start-create-transition-rule-desc
+
+Use the :mc-cmd:`mc ilm add` command to create a new transition rule
```

Comment:
This chunk gets included in all three of the transition tutorials as part of the step for creating the new transition rules.